x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/scalar/convert/ToString.java

```diff
@@ -221,4 +226,9 @@ static BytesRef fromGeoShape(BytesRef wkb) {
     static BytesRef fromGeoGrid(long gridId, @Fixed DataType dataType) {
         return new BytesRef(geoGridToString(gridId, dataType));
     }
+
+    @ConvertEvaluator(extraName = "FromExponentialHistogram")
+    static BytesRef fromExponentialHistogram(ExponentialHistogram histogram) {
+        return new BytesRef(exponentialHistogramToString(histogram));
```

Comment:
We convert to a string in exponentialHistogramToString (from a byte array), then convert to BytesRef here. However, since performance is not a concern for this method, it is perfectly fine to keep it this way instead of adding an exponentialHistogramToBytesRef method.